metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java

```diff
@@ -4084,6 +4084,28 @@ public void shouldTransformStringToBase64() {
         );
     }
 
+    @Test
+    public void shouldTransformUrlToBase64() {
```

Comment:
Maybe add another test for URL without `urlSafe`.

Reply:
Done with https://github.com/metafacture/metafacture-fix/pull/386/commits/5bebf359ec86c48c40528c0d2fadafdc7175a162 please have a look at it.

Reply:
Hm, I meant an actual assertion which simply verifies the non-URL encoded value.

Reply:
I thought it easier to see the difference. But I am also ok with you suggestion. Force pushed, see https://github.com/metafacture/metafacture-fix/pull/386/commits/d1498c0ca64bb6ed57a6d97c05101fc8dc04d9cf  .

Reply:
Interesting, that's exactly why I'm favouring the current setup - so you can actually see what the difference is :)